project-koku/koku
# COST-2357: Missing GCP data for the 1st day of the month.

Fixes [COST-2357](https://issues.redhat.com/browse/COST-2357)

Changes proposed in this PR:
* This pr is to remove a condition where we pass creating a file if the start & end date are the same. This condition was left over from the time we used to query usage dates in Bigquery. We now query for partition time so that condition is no longer needed because we can receive data that fits the time conditions in our bigquery query. 

Testing Instructions:
1. Download a real gcp source on main and look for `Manifest Data` in the logs:
```
koku-worker_1     | [2022-03-01 18:24:51,955] INFO a96d75ef-6a90-49d5-b606-c939ca66fead Manifest Data: {'assembly_id': 'b3d1a591-1078-494b-bc23-9d0fb9dcafb0:2183a92887c19bd1f9d083dabeb7e2f2:202203', 'compression': 'PLAIN', 'start_date': datetime.date(2022, 3, 1), 'end_date': datetime.date(2022, 3, 1), 'file_names': []}
```
Notice how the file_names is zero saying we downloaded no files for March.

Now switch over to this branch and add the same gcp source 
```
koku-worker_1     | [2022-03-01 19:30:34,003] INFO f73f0678-6066-4aa7-9bd8-34b803b44d98 Manifest Data: {'assembly_id': 'deac3997-6453-4710-81f9-a067e1d77d68:7a660d8b0e801b42f6b56859eeb7ae1c:202203', 'compression': 'PLAIN', 'start_date': datetime.date(2022, 3, 1), 'end_date': datetime.date(2022, 3, 1), 'file_names': ['202203_7a660d8b0e801b42f6b56859eeb7ae1c_2022-03-01:2022-03-02.csv']}
```
The file name is added and we see data for the GCP endpoint. 

---
Additional Context:
